cli/internal/cmd/iamcreateaws.go

```diff
@@ -26,11 +26,12 @@ func newIAMCreateAWSCmd() *cobra.Command {
 		RunE:  runIAMCreateAWS,
 	}
 
-	cmd.Flags().String("prefix", "", "Name prefix for all resources.")
+	cmd.Flags().String("prefix", "", "name prefix for all resources")
 	must(cobra.MarkFlagRequired(cmd.Flags(), "prefix"))
```

Comment:
```suggestion
	cmd.Flags().String("prefix", "", "name prefix for all resources (required)")
	must(cobra.MarkFlagRequired(cmd.Flags(), "prefix"))
```
I vote for marking required flags like this